### PR TITLE
VS-12128 Adding refusalReason for boleto

### PR DESCRIPTION
**Pull request by Nicole8164**

### CODE REVIEW

File: src/Message/AuthorizeResponse.php [hunk 1]
```diff
@@ -91,4 +91,14 @@ public function getDueDate()
     {
         return $this->data['additionalData_boletobancario_dueDate'];
     }
+
+    /**
+     * Returns the due date provided to the customer to complete boleto transaction
```
Comment (Dandroas) [inline src/Message/AuthorizeResponse.php]: This seems like it was copied and pasted.

Comment (Nicole8164) [inline src/Message/AuthorizeResponse.php]: AuthorizeReponse currently doesn't know about PaymentResponse which is where this is from. Do we want to create a trait for this to be shared across the two?

Comment (Nicole8164) [inline src/Message/AuthorizeResponse.php]: In PaymentResponse it's called getMessage() and currently AuthorizeResponse doesn't know about PaymentResponse. We could make it the same function name and make a trait to share between the two.

Comment (Dandroas) [inline src/Message/AuthorizeResponse.php]: I was referring to the docblock

Comment (Nicole8164) [inline src/Message/AuthorizeResponse.php]: Whoops fixing

**Pull request merged**